test/dotnet-remove-proj.Tests/GivenDotnetRemoveProj.cs

```diff
@@ -273,5 +316,82 @@ public void WhenPassedMultipleReferencesAndOneOfThemDoesNotExistItRemovesTheOneT
             slnFile.Projects.Count.Should().Be(1);
             slnFile.Projects[0].FilePath.Should().Be(Path.Combine("App", "App.csproj"));
         }
+
+        [Fact]
+        public void WhenReferenceIsRemovedBuildConfigsAreAlsoRemovedAndSlnBuilds()
```

Comment:
Similar feedback: this looks like it should be several tests with more specific names. 

The word "and" in a test name is usually a good indicator that you're testing more than one thing.